examples/droplet/main.tf

```
@@ -6,39 +15,40 @@ provider "digitalocean" {
 
 resource "digitalocean_droplet" "mywebserver" {
   # Obtain your ssh_key id number via your account. See Document https://developers.digitalocean.com/documentation/v2/#list-all-keys
-  ssh_keys           = [12345678]         # Key example
-  image              = "${var.ubuntu}"
-  region             = "${var.do_ams3}"
+  ssh_keys           = [29834845]        # Key example
```

This will allow the example to work with a real ssh key instead of hard coding an example key that wouldn't be on the user's account

```suggestion
  ssh_keys           = [digitalocean_ssh_key.example.fingerprint]  
```